scene/main/scene_tree.cpp

```diff
@@ -530,6 +530,7 @@ void SceneTree::process_timers(double p_delta, bool p_physics_frame) {
 		} else {
 			time_left -= p_delta;
 		}
+
```

Comment:
This change is unrelated and arguable. The `}` already acts as a separator, so empty line adds unnecessary space.

Reply:
Applied this suggestion and force-pushed.